src/views/donate-screen/index.js

```diff
@@ -1,24 +1,85 @@
 import { h } from "preact";
+import { useState, useEffect } from "preact/hooks";
 
 import Title from "../../components/title";
 import Button from "../../components/button";
-import { Content, Subtitle, Container, ContentWrapper } from "./styled";
 
-const DonateScreen = ({ onRequestClose, onNextClick, title }) => (
-  <Container>
-    <Title title={title} onRequestClose={onRequestClose} />
+import {
+  TagItem,
+  Content,
+  Subtitle,
+  Container,
+  CustomInput,
+  TagsWrapper,
+  ContentWrapper,
+} from "./styled";
 
-    <ContentWrapper>
-      <Subtitle>How much would you like to contribute?</Subtitle>
+const DONATION_AMOUNTS = [
```

Comment:
we should make the amounts and the currency configurable. with those values and EUR as defaults. 

and the text is always: `[VALUE] [CURRENCY]`

Reply:
yeah, that's doable. How do you want the currencies to be configured by the client? An array? something like:

```
{
...
currency: "EUR"/"DOLLAR" (or do you prefer symbols)?
paymentValueOptions: [1, 2, 3, 5, 10],
}
```

Reply:
something like that. 
can refactor it later. 
jfyi.

Reply:
sure, sounds good. I will go with this approach then.